content/docs/custom-plugins/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Installing Custom Plugins
+lastUpdated: '2021-03-16T17:55:00.0Z'
+description: How to add a custom Backstage plugin to Roadie
+---
+
+# Introduction
+
+Your plugins can be installed via npm or yarn, like publishing a normal package. We will provide you a unique scope for
+your plugins, and we host a dedicated npm registry which you can publish to.
+
+## Prerequisites
+
+- For our alpha release you'll need to give us some information upfront about your plugin. We plan to make all of this
+  self-service going forward but currently backstage doesn't support this out-of-the-box. For now, we will need you to
```

Comment:
I would blame Roadie for this rather than Backstage. It's unlikely Backstage will ever support this workflow.

```suggestion
self-service going forward but currently Roadie doesn't support this out-of-the-box. For now, we will need you to
```